testgrid/config.yaml

```diff
@@ -4697,6 +4683,25 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-kubeadm-gce-master
   - name: kubeadm-gce-stable-on-master
     test_group_name: ci-kubernetes-e2e-kubeadm-gce-stable-on-master
+
+- name: sig-release-master-informing
+  dashboard_tab:
```

Comment:
Side note: I need to go figure out what our process is again (talk to the release team?) But kind has been pretty stable on master for months now excluding actual Kubernetes breakages so I think I'd like to bring over the normal conformance suite from that too in this cycle. It's spent the last cycle being monitored by me for stability.

Reply:
seems fine, do it in a future PR?